docs/en/deployment.md

```diff
@@ -72,38 +99,82 @@ use the Region selector in the console navigation bar.
 
 You can view the status of the stack in the AWS CloudFormation Console in the **Status** column. You should receive a CREATE_COMPLETE status in approximately 10 minutes.
 
-### Step3: Update QuickSight permissions
+### Step2: Update QuickSight permissions
```

Comment:
why not moving it to prerequisites?

Reply:
need to select the S3 bucket which is created in deployment

Reply:
Any way to automation this step in deployment?

Reply:
I think we can moving this to prerequisites, see below doc,

https://docs.aws.amazon.com/quicksight/latest/user/scoping-policies-iam-interface.html

Reply:
moved it to prerequisites